src/config/localization/translations.json

```diff
@@ -1527,5 +1526,6 @@
   "Limit price is 12.12% above the current market rate. The order will be executed when the market price reaches the specified limit price within the expiration date, within your slippage parameters.": "Limit price is 12.12% above the current market rate. The order will be executed when the market price reaches the specified limit price within the expiration date, within your slippage parameters.",
   "Current Market Price": "Current Market Price",
   "Limit Price": "Limit Price",
-  "Error Tracking Id": "Error Tracking Id"
+  "Error Tracking Id": "Error Tracking Id",
+  "YOUR CAKE HELD AT BLOCK": "YOUR CAKE HELD AT BLOCK"
```

Comment:
Use lowercase here and make it uppercase in jsx

Reply:
thanks for reminding!